src/Logging/Logging.EventSource/src/EventSourceLogger.cs

```diff
@@ -205,19 +207,19 @@ private ExceptionInfo GetExceptionInfo(Exception exception)
 
         private string ToJson(IReadOnlyList<KeyValuePair<string, string>> keyValues)
         {
-            var sw = new StringWriter();
-            var writer = new JsonTextWriter(sw);
-            writer.DateFormatString = "O"; // ISO 8601
+            var arrayBufferWriter = new ArrayBufferWriter<byte>();
```

Comment:
Hmm... using an internal type, which wasn't meant to be used here. But hey, source package :)